extensions/wikia/CategorySelect/js/CategorySelect.view.js

```diff
@@ -77,7 +77,8 @@
 							controller: 'CategorySelectController',
 							data: {
 								articleId: articleId,
-								categories: $wrapper.data( 'categorySelect' ).getData( '.new' )
+								categories: $wrapper.data( 'categorySelect' ).getData( '.new' ),
+								token: mw.user.tokens.get('editToken')
```

Comment:
The previous one was simply throwing JS error in console and removing throbber so the UI is not blocked. I did the same with the new one. I'm open for better UX in this cases. We could discuss it with PO and propose the right copy + translations and some UI changes to display the message. Maybe it could be done as a separate story?